README.MD

```diff
@@ -1,48 +1,87 @@
-########################################################
-# 	Mantis Bugtracker Add-On
-# 	Query Version 2.13
-#	2012 - 2024 plugin by Cas Nuy www.NUY.info
-########################################################
+# Query plugin for MantisBT
 
-Query plugin
+Version 2.13
 
-This plugin will enable you to define multiple queries/scripts which:
-1. can be run online (by admin)
-2. can be scheduled in which case results are emailed to nominated recipients
+Copyright 2012 - 2024 by Cas Nuy www.NUY.info
+
+## Description
+
+This plugin will enable you to define multiple queries/scripts, which can be:
+1. run interactively (by admin)
+2. scheduled, in which case results are emailed to nominated recipients
+
+## Requirements
+
+The plugin requires MantisBT version 2.0.0 or higher.
+
+## Installation
 
-Installation
 As any other plugin, place the Query directory in your plugins directory.
-After installing it within Mantis, click on the module to finaliuze the settings.
+After installing it within Mantis, click on the module to finalize the settings.
+
+### PHP 8.2 
+
+When using PHP 8.2 or later, a change is required to `vendor/adodb/adodb-php/drivers/adodb-mysqli.inc.php`.
+Find the following lines
+
+```
+/*--------------------------------------------------------------------------------------
+	 Class Name: Recordset
+--------------------------------------------------------------------------------------*/
+class ADORecordSet_mysqli extends ADORecordSet{
+```
+and make it look like this:
+```
+/*--------------------------------------------------------------------------------------
+	 Class Name: Recordset
+--------------------------------------------------------------------------------------*/
+#[AllowDynamicProperties]
```

Comment:
Can you please explain why this is necessary ?

It seems strange to me, that you would need to add custom properties to the ADOdb RecordSet object, which is normally abstracted behind the MantisBT database API.